sessions/react-global-state/demo-end/pages/components/Counter.js

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  gap: 1ch;
+  align-items: center;
+
+  > :first-child {
+    min-width: 8ch;
+  }
+
+  > span {
+    min-width: 2ch;
+    text-align: center;
+  }
```

Comment:
We decided to not style child elements like this. We should create separated styled components here.